.github/workflows/pipeline.yml

```
@@ -29,6 +29,6 @@ jobs:
         java-version: '11'
         distribution: 'temurin'
     - name: Build with Gradle
-      uses: gradle/gradle-build-action@v2
+      uses: gradle/gradle-build-action@v2.1
```

...
suggestion
uses: gradle/gradle-build-action@v2.2.1
```suggestion
      uses: gradle/gradle-build-action@v2.1
```
...